sentinel5dl/__main__.py

```diff
@@ -141,14 +141,25 @@ def main():
             Example: 2019-09-17T23:59:59.999Z'''
     )
 
+    parser.add_argument(
+        '--useCertifi',
+        default='false',
+        action='store_true',
+        help='''If Certificate Authority (CA) bundle is not already supplied
+            by your operating system, certifi provides an easy way of
+            providing a cabundle.'''
+    )
+
     args = parser.parse_args()
 
+    # Search for Sentinel-5 products
     result = search(
         polygon=args.polygon,
         begin_ts=args.begin_ts,
         end_ts=args.end_ts,
         product=args.product,
-        processing_level=args.level
+        processing_level=args.level,
+        useCertifi=args.useCertifi
```

Comment:
CamelCase [is strongly discouraged for variables in Python by PEP8](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names). This should be called `use_certify` instead.

Reply:
ok. wondered why flake8 didnt complain then. it seems to be no option to enable it for flake8. using _pep8-naming_ in addition to flake8 now